mysql-test/suite/rocksdb.sys_vars/t/all_vars.test

```diff
@@ -1,3 +1,4 @@
+--source include/have_rocksdb_as_default.inc
```

Comment:
I was kind of thinking the same thing, it _does_ need RocksDB for the vars to be present in the first place to 'see' the vars, but no, it does not invoke RocksDB for the purposes of executing anything against it.